code/game/striketeams/striketeam_datums.dm

```diff
@@ -77,25 +77,32 @@ var/list/sent_strike_teams = list()
 	searching = TRUE
 
 	var/icon/team_logo = icon('icons/logos.dmi', logo)
-	for(var/mob/dead/observer/O in dead_mob_list)
-		if(!O.client || jobban_isbanned(O, ROLE_STRIKE) || O.client.is_afk())
-			continue
+	var/fake = FALSE//Antag Madness
```

Comment:
This var is necessary to track that this instance of a strike team request in particular was made before Antag Madness allows for ERT calls, and to prevent edge cases where the time limit end after the request was made but before the "ghost polling period" expires.

As for having a global antag_madness variable I think it saves some time during runtime instead of having to fetch the ruleset each time.

Reply:
Not going to ask you to refactor out of scope of the PR, but that seems silly to me. The name isn't very explicit either. As for the `antag_madness` var, access to the gamemode datum is negligible, but that's not really the place to ask you for this change.

Reply:
the var now uses some fairly explicit defines. please confirm if this will suffice for this PR.

Reply:
👍